opensrp-giz-malawi/src/main/java/org/smartregister/giz/processor/GizMalawiProcessorForJava.java

```diff
@@ -132,6 +133,11 @@ public void addMiniProcessors(MiniClientProcessorForJava... miniClientProcessors
         }
     }
 
+    @Override
+    public synchronized void processClient(List<EventClient> eventClientList, boolean localSubmission) throws Exception {
+        this.processClient(eventClientList);
+    }
+
```

Comment:
Because failing to explicitly specify this override results in code in core being executed instead of the client processor code.

Reply:
But haven't we already overridden the function that calls this. https://github.com/opensrp/opensrp-client-core/blob/41c86e4d09aaebd4e81730d5f4e5f847cf64a941/opensrp-app/src/main/java/org/smartregister/sync/ClientProcessorForJava.java#L81

Reply:
In core, NativeForm processor calls this method directly here ```org.smartregister.util.NativeFormProcessor#clientProcessForm```

Failing to override executes the method in core instead of the one implemented